bin/dependencies/generateDependencyReport.sh

```
@@ -26,11 +26,11 @@ echo "Build java report"
             tools:spring:spring-test-utilities:dependencies \
         >  bin/dependencies/${report_name}
     echo "  Java report for test app externalApp"
-    cd src/test/externalApp
-    ../../../gradlew dependencies >> ../../../bin/dependencies/${report_name}
+    cd test/externalApp
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)